tests-e2e/lists_test.go

```diff
@@ -1297,3 +1297,275 @@ func TestRangeOutsideTotalRooms(t *testing.T) {
 		),
 	)
 }
+
+// Nicked from Synapse's tests, see
+// https://github.com/matrix-org/synapse/blob/2cacd0849a02d43f88b6c15ee862398159ab827c/tests/test_utils/__init__.py#L154-L161
+// Resolution: 1×1, MIME type: image/png, Extension: png, Size: 67 B
+var smallPNG = []byte(
+	"\x89PNG\r\n\x1a\n\x00\x00\x00\rIHDR\x00\x00\x00\x01\x00\x00\x00\x01\x08\x06\x00\x00\x00\x1f\x15\xc4\x89\x00\x00\x00\nIDATx\x9cc\x00\x01\x00\x00\x05\x00\x01\r\n-\xb4\x00\x00\x00\x00IEND\xaeB`\x82",
+)
+
+func TestAvatarFieldInRoomResponse(t *testing.T) {
+	alice := registerNewUser(t)
+	bob := registerNewUser(t)
+	chris := registerNewUser(t)
+
+	t.Log("Alice, Bob and Chris upload an avatar.")
+	aliceAvatar := alice.UploadContent(t, smallPNG, "alice.png", "image/png")
+	bobAvatar := bob.UploadContent(t, smallPNG, "bob.png", "image/png")
+	chrisAvatar := chris.UploadContent(t, smallPNG, "chris.png", "image/png")
+
+	alice.SetAvatar(t, aliceAvatar)
+	bob.SetAvatar(t, bobAvatar)
+	chris.SetAvatar(t, chrisAvatar)
```

Comment:
Can we sanity check that all 3 point to different mxc URIs? Else this would rather break the tests / make them misleading.

Reply:
Enforced in 7ace2d9.